### PR TITLE
139/hotfix/update pagination props

### PR DESCRIPTION
**Pull request by mirovladimitrovski**
## OVERVIEW

Previously unused Pagination prop `currentPage` put into use for `react-paginate`:
`<ReactPaginate forcePage={currentPage} {...rest} />`

## WHERE SHOULD THE REVIEWER START?

_e.g. `/src/elements/Pagination/index.js`_

## CHECKLIST

_Be sure all items are_ ✅ _before submitting a PR for review._

- [ ] Verify the linters and tests pass: `yarn review`
- [ ] Verify you bumped the lib version according to [Semantic Versioning Standards](http://semver.org)
- [ ] Verify you updated the CHANGELOG
- [ ] Verify this branch is rebased with the latest master

### CODE REVIEW

File: src/elements/Pagination/index.js [hunk 1]
```diff
@@ -7,7 +7,6 @@ import PaginationWrapper from './PaginationWrapper';
 
 type Props = {
   pageCount: number,
-  currentPage: number,
```
Comment (srdjanRakic) [inline src/elements/Pagination/index.js]: I am quite sure that currentPage prop should not be deleted.

**Comment by srdjanRakic** [comment]
For every component change the snapshots should be updated.

File: src/elements/Pagination/index.js (hunk 1 above)
Comment (srdjanRakic) [inline src/elements/Pagination/index.js]: Remove currentPage prop from indexx.d.ts.

**Comment by mirovladimitrovski** [comment]
> Added forcePage prop is missing in index.d.ts.

**Comment by mirovladimitrovski** [comment]
> Added forcePage prop is missing in index.d.ts.

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,12 @@
 
 All notable changes to this project will be documented in this file.
 
+# [0.8.3] - 07-03-2019
+
+### Fixes
+
+- Put unused Pagination prop currentPage into use
```
Comment (andrej-naumovski) [inline CHANGELOG.md]: Please add a more descriptive changelog message, something like "Passed down currentPage prop to forcePage of ReactPaginate"

**Pull request merged**